### PR TITLE
manifest: update fw-nrfconnect-zephyr

### PR DESCRIPTION
**Pull request by rlubos**
Introduce recent upmerge.

Signed-off-by: Robert Lubos <robert.lubos@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: v1.14.99-ncs1
+      revision: pull/147/head
     - name: fw-nrfconnect-mcuboot
       path: mcuboot
       revision: v1.3.99-ncs1
```
Comment (SebastianBoe) [inline west.yml]: The mcuboot revision must contain https://github.com/NordicPlayground/fw-nrfconnect-mcuboot/pull/34

**Comment by SebastianBoe** [comment]
We need https://github.com/SebastianBoe/fw-nrfconnect-nrf-1/commit/228cf3f5ff72d86ae971992404f7a2a9caf471e4

File: west.yml [hunk 2]
```diff
@@ -40,10 +40,10 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 3c4f27282002e634d60216f23fb341317a4b199f
+      revision: pull/147/head
     - name: fw-nrfconnect-mcuboot
       path: mcuboot
-      revision: 4b919890cc848316304b81d9e160fe5a10ce3764
+      revision: pull/34/head
```
Comment (SebastianBoe) [inline west.yml]: We now have a SHA for this

Comment (rlubos) [reply west.yml]: Right, updated.

File: west.yml [hunk 3]
```diff
@@ -40,10 +40,10 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 3c4f27282002e634d60216f23fb341317a4b199f
+      revision: pull/147/head
```
Comment (mbolivar) [inline west.yml]: I've just +1ed this one

Comment (rlubos) [reply west.yml]: Thanks, FYI, we're waiting now for the nrf_desktop team to update their project and fix the CI, there were some API changes upstream (settings mostly).

**Comment by Qbicz** [comment]
@rlubos I have no permission to push to your fork. Can I get this or should I push elsewhere?

**Comment by rlubos** [comment]
@Qbicz Just pint me to your branch, I'll cherry-pick.

**Comment by Qbicz** [comment]
https://github.com/Qbicz/fw-nrfconnect-nrf/tree/upmerge-manifest

**Comment by rlubos** [comment]
@Qbicz Thanks, let's wait for the CI now.

**Comment by Qbicz** [comment]
Thanks @rlubos for updating manifest to account for mcuboot changes.

@pdunaj, can you review?

**Comment by Qbicz** [comment]
@kapi-no @grochu @KAGA164 Note that now the BLE services can be defined as static. This upmerge makes it work as before, but https://github.com/zephyrproject-rtos/zephyr/pull/15524/files introduces possibility to keep data structures in flash instead of RAM. Please review if some services in NCS can be made static (probably most of them apart from HID service).

**Pull request merged**